src/component/ResponsiveContainer.tsx

```diff
@@ -54,14 +54,19 @@ export const ResponsiveContainer = forwardRef(
       minHeight,
       maxHeight,
       children,
-      debounce = 0,
+      // debounce = 0,
       id,
       className,
       onResize,
       style = {},
-    }: Props,
+    },
     ref,
   ) => {
+    const containerRef = useRef<HTMLDivElement>(null);
+    const onResizeRef = useRef<Props['onResize']>();
+    onResizeRef.current = onResize;
+    useImperativeHandle(ref, () => containerRef.current);
```

Comment:
unfortunately this is a breaking change (though it is certainly a bug). See https://github.com/recharts/recharts/pull/2784

Reply:
If you want to merge, i can revert it